docs/digger/installation/macos-setup.asciidoc

```diff
@@ -1,46 +1,44 @@
-=== Setup macOS Servers
+[[macos-setup]]
+=== Setup macOS Servers (optional)
 
-==== Setting Ansible variables
-The macOS part of the installer will perform the following tasks.
+After setting the required Ansible variables, the installer will perform the following tasks to provision macOS servers:
 
-. Install Homebrew.
-. Install RVM and Ruby.
-. Install NVM and Node.
-. Install Xcode.
-. Download Apple certificates.
-. Update Cocoapods.
-. Configure machine as a node in Jenkins.
+. Install Homebrew,
```

Comment:
@pmacko1 not sure why we are using any punctuation here. I think with points there is no need for punctuation right?

Reply:
It depends. Here, I think you are correct.

Reply:
resolved